test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -214,6 +234,70 @@ void test_ucp_mmap::test_rkey_management(ucp_mem_h memh, bool is_dummy,
     ucp_rkey_buffer_release(rkey_buffer);
 }
 
+void test_ucp_mmap::compare_distance(const ucs_sys_dev_distance_t &dist1,
```

Comment:
Maybe `compare` is not such a good name for a function which is `void` (does not return a result, but checks/ensures values are equal)?